fragment/fragment/src/main/java/androidx/fragment/app/FragmentManager.java

```diff
@@ -2897,8 +2897,10 @@ void restoreSaveState(@Nullable Parcelable state) {
         mLaunchedFragments = new ArrayDeque<>(fms.mLaunchedFragments);
     }
 
+    /** @hide */
     @NonNull
-    FragmentHostCallback<?> getHost() {
+    @RestrictTo(RestrictTo.Scope.LIBRARY)
```

Comment:
Nit here and elsewhere: Always add the `@RestrictTo` annotation as the topmost annotation (i.e., directly below the Javadoc comment where the `@hide` is so that they are visually next to each other in the code).

Reply:
Makes sense - fixed in 4c0f698